tableauserverclient/models/target.py

```diff
@@ -0,0 +1,14 @@
+"""Target class meant to abstract mappings to other objects"""
+class Target():
+    def __init__(self, id, target_type):
+        self.id = id
+        self.target_type = target_type
+
+    def __repr__(self):
+        return "<Target#{id}, {target_type}>"
+
+    def get_type(self):
```

Comment:
I think I'm more of a fan of `type` rather than `target_type` because Target is already included in the name of the class, so it ends up being redundant.